frontend/src/app/sessions/user-sessions-wrapper/create-sessions/create-readonly-session/create-readonly-model-options/create-readonly-model-options.component.html

```diff
@@ -5,13 +5,22 @@
 
 <div [formGroup]="form">
   <h2>
-    <mat-checkbox color="primary" formControlName="include">{{
-      modelOptions.model.name
-    }}</mat-checkbox>
+    <mat-checkbox color="primary" formControlName="include"
+      >{{ modelOptions.model.name }}
```

Comment:
```suggestion
    <mat-checkbox color="primary" formControlName="include">
      {{ modelOptions.model.name }}
```
nit: These cases are not really caught by the autoformatter, but I still try to fix them when I see them.